crypto/src/merkle/merkle_path_set.rs

```diff
@@ -0,0 +1,357 @@
+use winter_utils::collections::BTreeMap;
+
```

Comment:
very minor nit: we can probably get rid of the blank line after this import.

Also, we should import `Vec` from winter_utils as well as we use them in this module. So, the import will look like:
```Rust
use winter_utils::collections::{BTreeMap, Vec};
```